docs/index.rst

```diff
@@ -1,8 +1,38 @@
-Welcome to gufe's documentation!
-================================
+gufe
+====
+
+**gufe** (grand unified free energy) is Open Free Energy's framework of object models used to describe alchemical free energy networks.
```

Comment:
Could I make a science request that we kill the idea that gufe is an acronym?

"grand" has a specific meaning in statistical mechanics and this isn't it.

Reply:
I always thought it was a joke, since "grand" does have a specific meaning, but AFAIK "unified" doesn't so I thought it was a riff on https://en.wikipedia.org/wiki/Grand_Unified_Theory 

Maybe we can pull a GROMACS and just change the acronym all the time -- Generalized Unified Free-energy Engineering is kinda fun

Reply:
Maybe General Underlying Free-energy Entities -- I went with graphical as is networks/graphs but that might imply a GUI

Reply:
I'll post all this on the issue you made since that is a better place to discuss it!